net/dns/resolver/tsdns.go

```diff
@@ -27,10 +28,19 @@ import (
 // truncation in a platform-agnostic way.
 const maxResponseBytes = 4095
 
-// queueSize is the maximal number of DNS requests that can await polling.
+// maxActiveQueries returns the maximal number of DNS requests that be running.
```

Comment:
missing a word ("can" ?)

Reply:
hah, yeah. Although I do like "that be runnin'".